src/Utilities/src/MoistThermodynamics.jl

```diff
@@ -411,10 +413,17 @@ The optional input value of the temperature `T_init` is taken as the initial
 value of the saturation adjustment iterations.
 """
 function saturation_adjustment(E_int, ρ, q_t, T_init = T_triple)
-
-    T = find_zero(x -> internal_energy_sat(x, ρ, q_t) - E_int, T_init,
-            Order1(), atol=1e-3*cv_d)
-
+    tol_abs = 1e-3*cv_d
+    iter_max = 100
```

Comment:
We should limit the number of iterations to < 10 or so.

Reply:
Ok, done (set `iter_max = 10`).